### DISCUSSION TITLE
EPS on 1 Sensor

### DISCUSSION THREAD

**Discussion by mtkhayes3** [Security-Onion-Solutions/securityonion#8322] [Unsupported Versions] [answered]
I have distributed set up air gapped and last week upgraded from 2.3.100 to 2.3.130.
We have manager/search and 10 sensor nodes.

The upgrade went great but 1 sensor is giving me issues.
I have not gotten any EPS and the Grid keeps going into fault for only the 1 sensor and then recovers.
Process Status: Fault Production and Consumption EPS is at 0.
Sensor so-status is all green except when the Grid goes into Fault so-status shows so-zeek in error and then clears???

I have reboot the Sensor.  It connects to Manager no issues.

**Comment by isaacgolding** [comment, marked as answer]
I'd start with zeek logs on the naughty sensor.    [From the SO docs:](https://docs.securityonion.net/en/2.3/zeek.html#diagnostic-logging)

Diagnostic Logging
Zeek diagnostic logs can be found in /nsm/zeek/logs/. Look for files like reporter.log, stats.log, stderr.log, and stdout.log. Depending on what you’re looking for, you may also need to look at the [Docker](https://docs.securityonion.net/en/2.3/docker.html#docker) logs for the container:

sudo docker logs so-zeek

**Reply by mtkhayes3** [reply]
Thank you for your help  xx Since it was only one sensor I was able to run setup .130 and issue was resolved.
EPS flowing in now.

**Comment by mtkhayes3** [comment]
Thank your for the help
